2-mandatory/2-water-bottle.js

```diff
@@ -18,19 +18,27 @@ You have to implement the missing features according to the specification.
 let bottle = {
   volume: 0,
   fillUp: function () {
-    // calling this function should completely fill your bottle (volume = 100);
+    return this.volume=100
   },
   pour: function () {
     // calling this function should increase your bottle volume by 10 units;
+    return this.volume + 10
```

Comment:
This will not reduce the `volume` by ten and it will only reduce the volume by 10 and return adjusted volume, so if you want to affect or change the original value of the volume in the object you need to do this I think `return this.volume = this.volume + 10`